components/connector-service/internal/externalapi/externalapi.go

```diff
@@ -15,48 +15,69 @@ import (
 	applicationSignatureHandler := NewSignatureHandler(appHandlerCfg.TokenManager, appHandlerCfg.CertService, appHandlerCfg.ContextExtractor)
 
-	applicationRouter := router.PathPrefix("/v1/applications").Subrouter()
-	httphelpers.WithMiddlewares(applicationRouter, appHandlerCfg.Middlewares)
-	applicationRouter.HandleFunc("/signingRequests/info", applicationInfoHandler.GetCSRInfo).Methods(http.MethodGet)
-	applicationRouter.HandleFunc("/certificates", applicationSignatureHandler.SignCSR).Methods(http.MethodPost)
+	csrApplicationRouter := router.PathPrefix("/v1/applications/signingRequests").Subrouter()
+	csrApplicationRouter.HandleFunc("/info", applicationInfoHandler.GetCSRInfo).Methods(http.MethodGet)
+
+	certApplicationRouter := router.PathPrefix("/v1/applications/certificates").Subrouter()
+	certApplicationRouter.HandleFunc("", applicationSignatureHandler.SignCSR).Methods(http.MethodPost)
+
+	httphelpers.WithMiddlewares(appHandlerCfg.Middlewares, csrApplicationRouter, certApplicationRouter)
+
+	mngmtApplicationRouter := router.PathPrefix("/v1/applications/management").Subrouter()
+	mngmtApplicationRouter.HandleFunc("/info", applicationManagementInfoHandler.GetManagementInfo).Methods(http.MethodGet)
 
-	runtimeInfoHandler := NewCSRInfoHandler(runtimeHandlerCfg.TokenManager, runtimeHandlerCfg.ContextExtractor, runtimeHandlerCfg.APIUrlsGenerator, runtimeHandlerCfg.CertificateURL, runtimeHandlerCfg.Subject)
+	httphelpers.WithMiddlewares(appMngmtInfoHandlerCfg.Middlewares, mngmtApplicationRouter)
+
+	runtimeInfoHandler := NewCSRInfoHandler(runtimeHandlerCfg.TokenManager, runtimeHandlerCfg.ContextExtractor, runtimeHandlerCfg.CertificateURL, runtimeHandlerCfg.GetInfoURL, runtimeHandlerCfg.ConnectorServiceHost, runtimeHandlerCfg.Subject, RuntimeURLFormat)
+	runtimeManagementInfoHandler := NewManagementInfoHandler(runtimeMngmtInfoHandlerCfg.ContextExtractor, runtimeMngmtInfoHandlerCfg.ConnectorServiceHost)
 	runtimeSignatureHandler := NewSignatureHandler(runtimeHandlerCfg.TokenManager, runtimeHandlerCfg.CertService, runtimeHandlerCfg.ContextExtractor)
 
-	runtimesRouter := router.PathPrefix("/v1/runtimes").Subrouter()
-	httphelpers.WithMiddlewares(runtimesRouter, runtimeHandlerCfg.Middlewares)
-	runtimesRouter.HandleFunc("/signingRequests/info", runtimeInfoHandler.GetCSRInfo).Methods(http.MethodGet)
-	runtimesRouter.HandleFunc("/certificates", runtimeSignatureHandler.SignCSR).Methods(http.MethodPost)
+	csrRuntimesRouter := router.PathPrefix("/v1/runtimes/signingRequests").Subrouter()
+	csrRuntimesRouter.HandleFunc("/info", runtimeInfoHandler.GetCSRInfo).Methods(http.MethodGet)
+
+	certRuntimesRouter := router.PathPrefix("/v1/runtimes/certificates").Subrouter()
+	certRuntimesRouter.HandleFunc("", runtimeSignatureHandler.SignCSR).Methods(http.MethodPost)
+
+	httphelpers.WithMiddlewares(runtimeHandlerCfg.Middlewares, csrRuntimesRouter, certApplicationRouter)
```

Comment:
Certificates generation for Runtime is not working, I guess there should be `certRuntimesRouter` instead of `certApplicationRouter`